0_config/code/nldas_cube_utils.R

```diff
@@ -0,0 +1,57 @@
+
+nldas_cube_diff <- function(new_cube_df, old_cube_df_filename){
+  old_cube_df <- readRDS(old_cube_df_filename)
+  diffed_cells <- dplyr::anti_join(new_cube_df, old_cube_df,  by = c('x','y')) 
+  
+  if (any(is.na(diffed_cells))){
+    stop('found NA(s) in NLDAS cell diff. Check ', old_cube_df_filename, ' and `new_cube_df` data')
+  }
+  cube_bounds <- diffed_cells %>% 
+    summarize(x0 = min(x), x1 = max(x), y0 = min(y), y1 = max(y))
```

Comment:
looks like this is just doing a spatial diff, right? makes sense for any early prototype, but just want to check - you're also planning to add a time component in the future, right?

Reply:
I can't seem to reply to the above comment about `I()`. I was going to hold off on that and see how my small example works in practice. I worry that adding some new systems and kicking off the rebuild may cause me to forget to `delete` that target and rebuild the diff config. Anyhow, yes, I am thinking about it, but not sure right now...

Reply:
and for time, yes, I admit I forgot I needed a time diff. But it is needed for efficiency.

Reply:
:+1:

Reply:
This gets more complicated, because I also need to include a _variable_ diff. Some variables may be complete for the cube, others not started, so working that up now and cleaning these other things up too. :+1: